scripts/helio/test_sync_noop_guards.py

```
@@ -0,0 +1,331 @@
+            return EXPR.sub(sub, body)  # no-op once every input arrives via env
+    raise SystemExit("step not found: " + name)
+
+
+CHECK = step_body("Check if already synced")
+GRAFT = step_body("Establish correct merge base (ephemeral graft)")
+
+
+def sh(cmd, cwd, env=None, check=True):
+    e = dict(os.environ)
+    e.update(env or {})
+    p = subprocess.run(["bash", "-c", cmd], cwd=cwd, env=e,
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)